loginpass/twitch.py

```diff
@@ -16,6 +16,17 @@
 from ._core import map_profile_fields
 
 
+def twitch_compliance_fix(session):
+
+    def fix_protected_request(url, headers, data):
+        headers["Client-ID"] = session.client_id
```

Comment:
Can you add twitch document link here as a comment.

Reply:
https://discuss.dev.twitch.tv/t/requiring-oauth-for-helix-twitch-api-endpoints/23916